server/app/controllers/api/v1/wishes_controller.rb

```diff
@@ -46,11 +45,15 @@ def wish_params
   end
 
   def set_api_user
```

Comment:
Why do we need the `api_user_id` param? I would expect that api user, would be `current_user`.

If we do need it, i think that logic should be moved to application_controller, so this controller doesn't need to be aware of this.

But I might be missing something

Reply:
Hi Dennis, if you are looking up a  user other then the current user than you can pass in the parameter in the url, and that parameter will be used to set the api_user

Reply:
If you are looking at profile the user will always be set to current_user, but if you are looking at api_user, wishes or experiences the parameter can be supplied, if not supplied it shows information for current user

Reply:
@dennis is this ok or are changes required?

Reply:
For experiences/wishes, there is always a API User provided in the URL?
https://github.com/RailsGirlsCPH/mentor-mentee-platform/blob/e2e790e957d98aa9799713548edf31ae0f59d225/server/config/routes.rb#L11-L18

Or can you leave it blank in order to use current_user?

Reply:
I have nothing else to add :)

Reply:
Thanks alot for reviewing Dennis